onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -1927,11 +1927,11 @@ static constexpr OrtApi ort_api_1_to_5 = {
     // End of Version 4 - DO NOT MODIFY ABOVE (see above text for more information)
 
     // Version 5 - In development, feel free to add/remove/rearrange here
-
     &OrtApis::GetStringTensorElementLength,
     &OrtApis::GetStringTensorElement,
     &OrtApis::FillStringTensorElement,
     &OrtApis::AddSessionConfigEntry,
+    &OrtApis::SetSessionProgrammingProjection,
```

Comment:
ahhh.. now i think i see.  version 5 is not complete yet ; and still under developent?